tabs/configuration.js

```diff
@@ -456,6 +456,10 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             serialRXtypes.push('Spektrum Bidir SRXL');
         }
 
+        if (semver.gte(CONFIG.flightControllerVersion, "3.2.0"))  {
```

Comment:
mr. norman explains why understanding, simplicity and transparency are crucial: http://amzn.to/2pbFmkW

as for the "api" nothing about msp protocol says it needs to be shared, that's a design decision and a poor reason to make the code more opaque